src/frontend/scss/colors/_theme.scss

```
@@ -247,6 +247,9 @@ $r-theme: (
     text-hover-color: r-color('grey59'),
     base-border: r-color('azure2'),
   ),
+  dashboard-course-list: (
+    title-color: r-color('firebrick6'),
```

I'll use something less saturate. In demo site, firebrick6 color acts as the primary color which should be keep for CTA.
```suggestion
    title-color: r-color('firebrick6'),
```

---

I would have used a grey color but let's start with this.